src/output/mod.rs

```diff
@@ -1,33 +1,79 @@
 //! Message output formats.
 
 mod dig;
+mod human;
+mod table;
+mod table_writer;
 
-
-use std::io;
-use clap::ValueEnum;
 use super::client::Answer;
+use clap::{Parser, ValueEnum};
+use domain::base::Ttl;
+use std::fmt::Write as _;
+use std::io;
+
+//------------ ANSI codes ----------------------------------------------------
```

Comment:
Maybe these should go into a separate `output::ansi` module? I would like to keep the actual code in mod.rs to an absolute minimum.

Also, then you can `use super::ansi` and have `ansi::BOLD` which I think makes it more clear what’s going on.

Reply:
I would agree in general, but because I use them in `format!` and that doesn't allow full paths yet, I still import them as `use super::ansi::BOLD`. I did put them in a separate module.

Reply:
Sounds good.